packages/injective-cosmwasm/src/query.rs

```diff
@@ -38,6 +38,21 @@ pub enum InjectiveQuery {
         market_id: String,
         subaccount_id: String,
     },
+    TraderSpotOrdersToCancelUpToAmount {
+        market_id: String,
+        subaccount_id: String,
+        base_amount: FPDecimal,
+        quote_amount: FPDecimal,
+        strategy: i32,
+        reference_price: Option<FPDecimal>,
+    },
+    TraderDerivativeOrdersToCancelUpToAmount {
+        market_id: String,
+        subaccount_id: String,
+        quote_amount: FPDecimal,
+        strategy: i32,
```

Comment:
we should have some documentation (or possibly an enum) on the cancellation strategy options

Reply:
https://github.com/InjectiveLabs/cw-injective/issues/52